src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -50,7 +54,16 @@ function NameAndPicture({ navigation, route }: Props) {
   const asyncKomenciReadiness = useAsyncKomenciReadiness()
   const showGuidedOnboarding = useSelector(showGuidedOnboardingSelector)
   const createAccountCopyTestType = useSelector(createAccountCopyTestTypeSelector)
-  const skipUsername = route.params?.skipUsername //TODO repalce with statsig variable
+  const skipUsername = shouldSkipUsername() //TODO repalce with statsig variable
+  const showAlternatePlaceholder = shouldShowAlternatePlaceholder() //TODO repalce with statsig variable
+  let placeholderUsername = t('fullNamePlaceholder')
+  if (showGuidedOnboarding) {
```

Comment:
Is this coming from an existing firebase experiment / remote config? Wonder if we could use a "usernamePlaceholder" parameter in statsig, which can then take values of "fullName", "fullNameOrPseudoym", or "alternate" (or just "cyrptoAlterEgo")?

Reply:
Yes, but the `showGuidedOnboarding` has already started on Firebase and I'd prefer letting it run; there was discussion that ended in letting that finish before starting the `cyrptoAlterEgo` experiment on StatSig.

Reply:
yes that sounds good. I am just proposing that the new flag we add to statsig is an enum instead of a boolean. As @jh2oman mentioned, looks like we already have `nameType`. We should just be using that instead of a new boolean flag. Eventually the code can just become something like.
```js
const nameType = Statsig.get(....) // get param value from statsig
switch(nameType) {
  case 'first_and_last':
    placeholderUserName = t('fullName')
  case 'first_and_last_or_pseudonym':
    placeholderUserName = t('fullNameOrPseudonym')
  case 'crypto_alter_ego':
    placeholderUserName = 'MyCryptoAlterEgo'
}
```
One complication here is that we have to keep honoring the guided onboarding flag until that experiment is complete. Maybe we can add a 4th case temporarily as the default value of the param e.g., 
```js
  case 'guided_onboarding_exp':
    placeholderUserName =  showGuidedOnboarding ? t('fullNameOrPseudonym') :  t('fullName')
```
and once this experiment is over, depending on whichever side wins, we can make  `first_and_last` or `first_and_last_or_pseudonym` the default value